trade-finance-manager-ui/component-tests/case/facility/_macros/facility_details.component-test.js

```diff
@@ -169,6 +180,78 @@ describe(component, () => {
     it('should render tenor', () => {
       wrapper.expectText('[data-cy="facility-tenor"]').toRead(params.facility.dates.tenor);
     });
+
+    describe('when facility end date feature flag is not enabled', () => {
+      beforeEach(() => {
+        wrapper = render({ ...params, facility: facilityWithFacilityEndDate, isTfmFacilityEndDateFeatureFlagEnabled: false });
+      });
+      it('should not render isUsingFacilityEndDate', () => {
+        wrapper.expectText('[data-cy="facility-end-date"]').notToExist();
+      });
+
+      it('should not render facilityEndDate', () => {
+        wrapper.expectText('[data-cy="facility-end-date"]').notToExist();
+      });
+
+      it('should not render bankReviewDate', () => {
+        wrapper.expectText('[data-cy="bank-review-date"]').notToExist();
+      });
+    });
+
+    describe('when facility end date feature flag is enabled', () => {
+      describe('when the facility has no end date information', () => {
+        beforeEach(() => {
+          wrapper = render({ ...params, isTfmFacilityEndDateFeatureFlagEnabled: true });
+        });
+        it('should render a dash for isUsingFacilityEndDate', () => {
+          wrapper.expectText('[data-cy="facility-end-date-exists"]').toRead('-');
+        });
+
+        it('should render a dash for facilityEndDate', () => {
+          wrapper.expectText('[data-cy="facility-end-date-exists"]').toRead('-');
```

Comment:
same here
Also noticed this uses the old name for `isUsingFacilityEndDate` - think it should be `is-using-facility-end-date`